ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1200,6 +1200,85 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-azure-upi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-baremetalds
```

Comment:
Can we make the name that's visible to users something like `e2e-metal-ipi` ?  I don't think `baremetalds` is going to be meaningful to most people